config/jobs/kubernetes/kops/kops-periodics-network-plugins.yaml

```diff
@@ -194,6 +258,131 @@ periodics:
     testgrid-days-of-results: '90'
     testgrid-tab-name: kops-aws-cni-cilium
 
+# {"cloud": "azure", "distro": "u2404", "feature_flags": "Azure", "k8s_version": "stable", "kops_channel": "alpha", "kops_version": "latest", "networking": "cilium"}
+- name: e2e-kops-azure-cni-cilium
```

Comment:
I think this is replacing `e2e-kops-k8s-azure-cilium` ? (drop it in the same PR?)

Reply:
+1